### ISSUE TITLE
Support for Yahoo Accounts

### ISSUE THREAD

**Issue by andrewlai87** [rustyio/super-imap#6]
Looking at this library, it seems like it utilizes IMAP-Idle.  However, Yahoo doesn't support IDLE, so I was wondering, does this service work for Yahoo accounts?

**Comment by rustyio** [comment]
No, it likely doesn't work with Yahoo accounts. The design relies heavily on the presence of IDLE support. 

Unfortunately, it's unlikely that I'll make any changes or improvements to this project. I've left the company using this code, and have moved on to another venture. It's best to consider the existing codebase stable but -- as of now -- unmaintained.
